### PR TITLE
Corrected casing of words on Monthly GST report page

### PR DESCRIPTION
**Pull request by NirmalNegi07**
Targets #1684 
### Description
Corrected casing of words on Monthly GST report page

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: Modules/Invoice/Resources/views/monthly-gst-report.blade.php [hunk 1]
```diff
@@ -26,12 +26,12 @@
                     <th>Date</th>
                     <th>Particular</th>
                     <th>Type</th>
-                    <th>INVOICE NO.</th>
+                    <th>Invoice No.</th>
```
Comment (rathorevaibhav) [inline Modules/Invoice/Resources/views/monthly-gst-report.blade.php:29]: @NirmalNegi07 this is all good. can you also update the heading for this page? It's not mention in the issue but I just noticed it.

From `Monthly Gst Report` to `Monthly GST Report`

Comment (NirmalNegi07) [reply Modules/Invoice/Resources/views/monthly-gst-report.blade.php:29]: Ohk sir

**Comment by NirmalNegi07** [comment]
kindly review my task

**Review by P4NK4J — APPROVED** [review]
looks good

**Pull request merged**